### PR TITLE
Add standard files

### PR DESCRIPTION
**Pull request by zippolyte**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+| `kind/bug`                    | Issue Only               | Bug related issue.
+| `kind/feature-request`        | Issue Only               | Feature request related issue.
+| `severity/critical`           | Issue Only               | Critical severity issue.
+| `severity/major`              | Issue Only               | Major severity issue.
+| `severity/minor`              | Issue Only               | Minor severity issue.
+| `severity/normal`             | Issue Only               | Normal severity issue.
+| `stale`                       | Issues and Pull Requests | Stale - Bot reminder.
+| `stale/exempt`                | Issues and Pull Requests | Exempt from being marked as stale.
+
+
```
Comment (gzussa) [inline CONTRIBUTING.md]: remove line breaks

File: LICENSE-3rdparty.csv [hunk 2]
```diff
@@ -0,0 +1 @@
+Component,Origin,License,Copyright
```
Comment (gzussa) [inline LICENSE-3rdparty.csv:1]: Is this empty on purpose?

**Review by gzussa — CHANGES_REQUESTED** [review]
Let's add the support file. We should add a release instruction. Even if it is to say that it will be dealt by us.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+Please don't be this person: `git commit -m "Fixed stuff"`. Take a moment to
+write meaningful commit messages.
+
+The commit message should describe the reason for the change and give extra details
+that will allow someone later on to understand in 5 seconds the thing you've been
+working on for a day.
+
+### Releasing
+
+The release procedure is managed by Datadog.
```
Comment (gzussa) [inline CONTRIBUTING.md]: Just saw this. I think we should have a file to explain that we would make a release pretty much at the same pace as the agent release.

File: LICENSE-3rdparty.csv (hunk 2 above)
Comment (zippolyte) [reply LICENSE-3rdparty.csv:1]: yeah

File: RELEASING.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Releasing
+
+Release can only be performed by Datadog maintainers of this repository.
+
+## Schedule
+
+Releases of this project follow the [Datadog Agent](https://github.com/DataDog/datadog-agent) release schedule of one minor release every 6 weeks.
```
Comment (gzussa) [inline RELEASING.md]: Let's change it to 2 months

**Review by gzussa — CHANGES_REQUESTED** [review]

File: RELEASING.md (hunk 4 above)
Comment (zippolyte) [inline RELEASING.md]:
```suggestion
Releases of this project follow the [Datadog Agent](https://github.com/DataDog/datadog-agent) release schedule (+1/2 weeks).
```

**Review by gzussa — APPROVED** [review]

**Pull request merged**